packages/web-components/fast-components/src/avatar/avatar.styles.ts

```diff
@@ -0,0 +1,68 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "@microsoft/fast-foundation";
+import { accentFillRestBehavior, neutralForegroundRestBehavior } from "../styles/index";
+
+export const AvatarStyles = css`
+    ${display("flex")} :host {
+        max-width: var(--avatar-size, var(--avatar-size-default));
+        height: var(--avatar-size, var(--avatar-size-default));
+        position: relative;
+        --avatar-size-default: calc(
+            (
+                    (var(--base-height-multiplier) + var(--density)) * var(--design-unit) +
+                        (var(--design-unit) * 10)
```

Comment:
This seems very large. I'm working on adding the designs for this now, but until we land a direction where this should be much larger can we make this the standard height (which will end up 32px with default values)?

Reply:
@bheston do you have additional thoughts or ideas on this?

Reply:
> This seems very large. I'm working on adding the designs for this now, but until we land a direction where this should be much larger can we make this the standard height (which will end up 32px with default values)?

Could you clarify this? Are you suggesting that the default size be set to 32px?